ts/parts/Pointer.ts

```diff
@@ -2106,51 +2127,83 @@ class Pointer {
             ownerDoc = container.ownerDocument;
 
         container.onmousedown = function (e: MouseEvent): void {
-            pointer.onContainerMouseDown(e as any);
+            pointer.onContainerMouseDown(pointer.normalize(e));
```

Comment:
`Pointer.onContainerMouseDown` already does `normalize` internally...

Reply:
That is a type thing, because I liked to remove the `any` casts. Without normalize `onContainerMouseDown` has to accept regular `MouseEvent`s without normalizing. It is the same with the other pointer callbacks.